fastlane/Fastfile

```diff
@@ -0,0 +1,24 @@
+lane :test_bump_version_update_changelog_create_pr do |options|
```

Comment:
Is this just to test them locally in the repo?

Reply:
These are mostly as an example of how to call the different actions. If we currently call them, they will fail, since the parameters passed do not match the project setup. I think I can rename these lanes to `sample_bump_version_update_changelog_create_pr_call` or something like that